spec/models/message_spec.rb

```diff
@@ -1,38 +1,31 @@
 require 'rails_helper'
 
-describe Message do
+RSpec.describe Message, type: :model do
```

Comment:
An RSpec file in the `spec/models` directory is automatically of `type: :model`.
Rspec can be configured to infer the spec type by the location of the faile, you can implement this using: 
```
​# spec/rails_helper.rb
RSpec.configure do |config|
  config.infer_spec_type_from_file_location!
end
```
You can take a look at [the documentation](https://relishapp.com/rspec/rspec-rails/docs/directory-structure)

Reply:
Yeah, actually i had it that way, but i think i added it trying to fix some error (at the end it was not releated) but forgot to change it, thanks Alvaro! :raised_hands: